geonetwork/gn-tool/gn3-tool.sh

```diff
@@ -58,8 +58,11 @@ get_all_records() {
     if [[ $gn_xsrftoken != "" ]]; then
       gn_xsrftoken_arg="--xsrftoken $gn_xsrftoken"
     fi
+    if [[ $harvested_records == "" ]]; then
```

Comment:
For the prod migration, we now want to be able to harvest all records, harvested and not harvested.  Will this mean we have to run the export twice now, once for harvested and another time for non-harvested?